Jenkinsfile

```diff
@@ -0,0 +1,25 @@
+pipeline {
+  agent {
+    docker {
+      image 'maven:3.5.2-jdk-8'
+    }
+  }
+  environment {
+      tomcatCredentials = credentials('TOMCATCREDENTIALS')
+      developmentHost = credentials('DEVELOPMENTHOST')
+    }
+  stages {
+    stage('Build') {
+      steps {
+        sh 'mvn clean -Daccession.service=file-accession install -DskipTests -DbuildDirectory=file/target'
```

Comment:
As I also said in other comments, we only want to have one application, not multiple instances. Please, unify the execution in one application, change deployment script apropriatedly.